### PR TITLE
Fix issues in stolon chart introduced during the migration

### PR DESCRIPTION
**Pull request by lwolf**
**What this PR does / why we need it:**
* Add missing emptyDir volume
* Add missing serviceAccountName

**Which issue this PR fixes**
Broken install if persistence is disabled

### CODE REVIEW

**Comment by lwolf** [comment]
/assign @prydonius 
The bot told me assign it to you, sorry for any inconveniences.

File: stable/stolon/templates/proxy-deployment.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ spec:
 {{ toYaml . | indent 8 }}
 {{- end }}
     spec:
+      serviceAccountName: {{ template "stolon.serviceAccountName" . }}
```
Comment (lachie83) [inline stable/stolon/templates/proxy-deployment.yaml:29]: Should you set the value of serviceAccountName to default in values.yaml? - https://github.com/kubernetes/charts/blob/master/stable/stolon/values.yaml#L33

Comment (lwolf) [reply stable/stolon/templates/proxy-deployment.yaml:29]: the name will be generated based on fullname.
I think behaviour is quite clear from the comment in values.yaml:
```
# The name of the ServiceAccount to use. If not set and create is true, a name is generated using the fullname template
```

Comment (lachie83) [reply stable/stolon/templates/proxy-deployment.yaml:29]: My apologies. This was misread on my side.

**Pull request merged**